cns/hnsclient/hnsclient_windows.go

```diff
@@ -490,7 +524,8 @@ func CreateHostNCApipaEndpoint(
 	networkContainerID string,
```

Comment:
you need to update the func definition signature in hnsclient_linux.go too.

Reply:
Updated